shared/tsconfig.json

```diff
@@ -31,6 +31,7 @@
     "esModuleInterop": true,
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
-    "composite": true
+    "composite": true,
+    "skipLibCheck": true
```

Comment:
needed to solve a TS error with Drizzle ORM (suspect it's to do with diverging TS version?)